apps/i3wm/i3wm.talon

```diff
@@ -2,82 +2,81 @@
+focus floating: user.i3wm_focus("mode_toggle")
+center window: user.i3wm_move_position("center")
+resize mode: user.i3wm_mode("resize")
+focus parent: user.i3wm_focus("parent")
+focus child: user.i3wm_focus("child")
 
 # resize helpers
 grow window:
-    user.system_command('i3-msg mode "resize"')
+    user.i3wm_mode("resize")
     key(right:10)
     key(down:10)
     # escape resize mode
     key(escape)
     # center window
     sleep(200ms)
-    user.system_command("i3-msg move position center")
+    user.i3wm_move_position("center")
 
 # resize helpers
 shrink window:
-    user.system_command('i3-msg mode "resize"')
+    user.i3wm_mode("resize")
     key(left:10)
     key(up:10)
     # escape resize mode
     key(escape)
     # center window
     sleep(200ms)
-    user.system_command("i3-msg move position center")
+    user.i3wm_move_position("center")
 
 horizontal (shell | terminal):
-    user.system_command("i3-msg split h")
+    user.i3wm_split("h")
     user.i3wm_shell()
 
 vertical (shell | terminal):
-    user.system_command("i3-msg split v")
+    user.i3wm_split("v")
     user.i3wm_shell()
 
 # XXX - just replace with shuffle eventually?
 # XXX - like also need to match the generic talon commands
 (shuffle | move (win | window) [to] port) <number_small>:
-    user.system_command("i3-msg move container to workspace number {number_small}")
+    user.i3wm_move_to_workspace(number_small)
 (shuffle | move (win | window) [to] last port):
-    user.system_command("i3-msg move container to workspace back_and_forth")
-(shuffle | move) flipper:
-    user.system_command("i3-msg move container to workspace back_and_forth")
-(shuffle | move (win | window) left): user.system_command("i3-msg move left")
-(shuffle | move (win | window) right): user.system_command("i3-msg move right")
-(shuffle | move (win | window) up): user.system_command("i3-msg move up")
-(shuffle | move (win | window) down): user.system_command("i3-msg move down")
+    user.i3wm_move_to_workspace("back_and_forth")
+(shuffle | move) flipper: user.i3wm_move_to_workspace("back_and_forth")
+(shuffle | move (win | window) left): user.i3wm_move("left")
+(shuffle | move (win | window) right): user.i3wm_move("right")
+(shuffle | move (win | window) up): user.i3wm_move("up")
+(shuffle | move (win | window) down): user.i3wm_move("down")
 
-(win | window) horizontal: user.system_command("i3-msg split h")
-(win | window) vertical: user.system_command("i3-msg split v")
+(win | window) horizontal: user.i3wm_split("h")
+(win | window) vertical: user.i3wm_split("v")
 
-make scratch: user.system_command("i3-msg move scratchpad")
-[(show | hide)] scratch: user.system_command("i3-msg scratchpad show")
+make scratch: user.i3wm_move("scratchpad")
+[(show | hide)] scratch: user.i3wm_show_scratchpad()
 next scratch:
-    user.system_command("i3-msg scratchpad show")
-    user.system_command("i3-msg scratchpad show")
+    user.i3wm_show_scratchpad()
+    user.i3wm_show_scratchpad()
```

Comment:
`next scratch` doesn't do the right thing on my computer (it just swaps in and out of scratch mode). But I think we can just keep this PR as about refactoring the code, not fixing bugs.